htdocs/map.js

```diff
@@ -37,6 +37,7 @@ $(function(){
     var retention_time = 2 * 60 * 60 * 1000;
     var strokeOpacity = 0.8;
     var fillOpacity = 0.35;
+    var callsign_url = null;
```

Comment:
I'm assuming that the "unconfigured" value would be an empty string. If so, it's probably best to set this to an empty string, too.

Reply:
As I previously mentioned, set it to anything you like.

Reply:
Not necessarily talking about the default value here. Just trying to simplify the code: If this was an empty string from the beginning, you would not need to test for `null` later on (line 348 RN)

Reply:
Oh, that. I always test optional pointers for null. If you are only testing for empty string, you are setting yourself up for a crash in the future.

Reply:
This is javascript, I don't think the concept of pointers exists here, at least not in the code.

Either way, I don't mind the extra check, it certainly won't hurt.